conf/location-autocomplete-canonical-list.json

```diff
@@ -1,280 +1,236 @@
 [
   [
-    "Abu Dhabi",
-    "territory:AE-AZ"
+    "Andorra",
+    "country:AD"
+  ],
+  [
+    "United Arab Emirates",
+    "country:AE"
   ],
   [
     "Afghanistan",
     "country:AF"
   ],
   [
-    "Ajman",
-    "territory:AE-AJ"
+    "Antigua and Barbuda",
+    "country:AG"
   ],
   [
-    "Akrotiri",
-    "territory:XQZ"
+    "Anguilla",
+    "country:AI"
   ],
   [
     "Albania",
     "country:AL"
   ],
   [
-    "Algeria",
-    "country:DZ"
-  ],
-  [
-    "American Samoa",
-    "territory:AS"
-  ],
-  [
-    "Andorra",
-    "country:AD"
+    "Armenia",
+    "country:AM"
   ],
   [
-    "Angola",
+    "Angola including Cabinda",
```

Comment:
Wow!
That seems a bit harsh.
This is a massive political statement.
A bit of political context here:
Cabinda is an exclave and province of Angola, surrounded by Democratic Republic of the Congo, they are the richest province of Angola and are responsibles for the majority of oil production.
They are effectively the motor of the Angolan economy and have suggested in the past the will to become independent, although that would probably cause a civil war followed by a war with DRC.

It is the first time I see mentioned `Angola including Cabinda`